assets/js/pages/Users/UsersPage.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { listUsers, deleteUser } from '@lib/api/users';
+import { format, parseISO } from 'date-fns';
+
+import { toast } from 'react-hot-toast';
+
+import Users from './Users';
+
+function UsersPage() {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [deleteUserId, setDeleteUserId] = useState(null);
```

Comment:
Like said before, the visual thing of the inner component should be in the inner component.

We should follow a dummy/container components approach


This UsersPage is the container, that contains logic, external dependencies, api call, the inner component is dummy, but controlled in it's visual things and use the prop passed from outside just for side effects, like api calls